### ISSUE TITLE
batch normalization layer which translated by toco cannot be run in TF Lite

### ISSUE THREAD

**Issue by Corea** [tensorflow/tensorflow#15336]
## Issue
Resolver for batch normalization layer(`ResolveBatchNormalization`) in toco has four inputs, which are input matrix, mean, multiplier, and offset respectively. Usually, input matrix has four dimensions (NHWC) while mean, multiplier, and offset only have 1 dimension for a channel. This resolver translates this 1 dimension vector to second input of `Mul` and `Add` layers in TF Lite. (first input of these layers is input matrix!) But the problem occurs `Mul` and `Add` layers in TF Lite have a process to check dimension of inputs. Need to fix this contradiction. :) 

## Codes related to this Issue
[Resolver](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/toco/graph_transformations/resolve_batch_normalization.cc)
[Logic of checking dimension in add layer](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/kernels/add.cc#L48)
[Logic of checking dimension in mul layer](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/kernels/mul.cc#L48)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue? Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by reedwm** [comment]
Closing because the issue template was never filled out. Please reopen and fill out the template if this is still a problem.

**Comment by milinddeore** [comment]
I am giving #[23627](https://github.com/tensorflow/tensorflow/issues/23627)  reference of the other issue that i opened up in the same regard, but please reopen this issue to address the concern raised. I have given all the details in the templet required. Thanks.
